packages/report-e2e-tests/src/examples/combined-results-with-issues.snap.html

```diff
@@ -616,110 +683,18 @@
       .results-container {
         margin-top: 56px;
       }
-      .urls-summary-section {
```

Comment:
You'll notice this style is removed but not added anywhere in this PR with the hash:

This is because a previous PR already modified the related component to use css-modules. This "removal" is just removing the import of the scss file (as a result of removing summary-report.scss completely).